### DISCUSSION TITLE
Ability to sort timers

### DISCUSSION THREAD

**Discussion by Kritoris** [smasherprog/EqTool#134] [Ideas]
So typically, the way my brain works is to have the most urgent things at the "top of the list".

Currently, mob spawn timers are top to bottom, last to spawn down to closest to spawn which feels like reverse order.

Then, mob debuffs/PC Buffs seem to be in "order of cast", so again a little harder to see what is the most "top" of priority as there is no order of expiry.

So, it would be good to be able to sort in different manners. eg. Soonest to expire, last to expire or order cast/killed etc.

Thoughts?

**Comment by smasherprog** [comment]
This all sounds correct to me. Ill add it to the to do list

**Comment by smasherprog** [comment]
Changed timers default sort order to be Least amount of time left to most amount of time left.
